assets/js/components/module/ModuleApp.js

```diff
@@ -68,20 +68,24 @@ function ModuleApp( { moduleSlug } ) {
 				{ moduleConnected && <DateRangeSelector /> }
 			</Header>
 			<Alert module={ moduleSlug } />
-			{ shouldRenderWidget &&
-				<Fragment>
-					<WidgetContextRenderer
-						slug={ screenWidgetContext }
-						className={ classNames( [
-							'googlesitekit-module-page',
-							`googlesitekit-module-page--${ moduleSlug }`,
-						] ) }
-						Header={ getModuleHeader }
-						Footer={ ModuleFooter }
-					/>
-				</Fragment>
+			{ shouldRenderWidget
+				? (
+					<Fragment>
+						<WidgetContextRenderer
+							slug={ screenWidgetContext }
+							className={ classNames( [
+								'googlesitekit-module-page',
+								`googlesitekit-module-page--${ moduleSlug }`,
+							] ) }
+							Header={ getModuleHeader }
+							Footer={ ModuleFooter }
+						/>
+					</Fragment>
+				)
+				: (
+					<LegacyModuleApp />
+				)
```

Comment:
Nit: We usually don't wrap elements that fit on a single line, but instead use it when the element spans multiple lines so that open and closing symbols are in the same vertical column. No big deal though 😄